sass-msg-formatter.js

```diff
@@ -0,0 +1,105 @@
+const chalk = require('chalk');
+
+// colored text constants
+const errorText = chalk.bold.red;
+const warningText = chalk.yellow;
+const urlText = chalk.underline.cyan;
+const titleText = chalk.bgYellow;
+const numberedText = chalk.dim;
+const ruleNameText = chalk.bgWhite.black;
+
+/*
+ * filterForErrors: filters out elements in the results array that don't have errors
+ */
+function filterForErrors(result) {
+  return result.errored;
+}
+
+/*
+ * generateErrorIcon: generates a custom error or warning text depending on the severity of the error
+ */
+function generateErrorIcon(severity) {
```

Comment:
could you add test coverage for this? 
could you add jsdocs for the param?

Reply:
jsdoc was added to the `sassMsgFormatter`, this duplicate file was deleted